docs/csslint-examples.md

```diff
@@ -59,6 +59,26 @@ csslint: {
 }
 ```
 
+## Providing a custom formatter
+
+```js
+csslint: {
+  options: {
+    csslintrc: '.csslintrc',
+    formatter: require('csslint-stylish),
+    formatters: [
+      {id: 'stylish', dest: 'report/csslint_stylish.xml'}
```

Comment:
After thinking about this for a bit, it would probably be better to just have `id` accept the required formatter in addition to a string. That way we don't have to have a separate option.

```js
formatters: [
      {id: require('csslint-stylish), dest: 'report/csslint_stylish.xml'}
]
```